bark_core/signatures.py

```diff
@@ -386,6 +388,14 @@ def get_authorized_pubkeys(
 def get_pubkey_from_git() -> Optional[Pubkey]:
     identifier = cmd("git", "config", "user.signingKey", check=False)[0]
     if identifier:
+        gpg_format = cmd("git", "config", "gpg.format", check=False)[0]
+        # check if signingKey is ssh
+        if gpg_format and gpg_format == "ssh":
+            email = cmd("git", "config", "user.email", check=False)[0]
+            identifier = os.path.expanduser(identifier) # expand path if relative
+            with open(identifier, 'r') as file: # read public key
```

Comment:
Change file reading mode from 'r' to 'rb'.

Reply:
Done 😊

Reply:
You will have to remove `pubkey.encode()`, since pubkey is now already in `bytes`.